test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -272,5 +272,17 @@ Describe "Get-Help should find pattern help files" -Tags "CI" {
         )
         $command.Invoke() | Should Be $result
     }
+}
+
+  Describe "Get-Help should find pattern alias" -Tags "CI" {
+    
+    It "Get-Help should find alias as command" {
+       (Get-Help where).Name | Should be "Where-Object"
+    }
+
+    It "Get-Help should find alias with ? pattern" {
+       (Get-Help wher?).Category | Should be "Alias"
```

Comment:
You can store the output of ```Get-Help wher?``` into a variable and then check for Category and Synopsis.

Reply:
resolved